### DISCUSSION TITLE
「再添一砖」Memos 浏览器扩展分享

### DISCUSSION THREAD

**Discussion by lmm214** [usememos/memos#274] [Ideas]
项目地址： [https://github.com/lmm214/memos-bber](https://github.com/lmm214/memos-bber)

简单教程：[https://immmmm.com/chrome-extensions-memos-bber/](https://immmmm.com/chrome-extensions-memos-bber/)

**Comment by neilwyh** [comment]
不错，已经用上了
可以在chrome上面快速记录
